assets/app/scripts/directives/overviewDeployment.js

```diff
@@ -27,27 +28,39 @@ angular.module('openshiftConsole')
           $scope.desiredReplicas = null;
         });
 
-        // Debounce scaling so multiple clicks within 500 milliseconds only
-        // result in one request.
+        $scope.canScale = function() {
+          if (!$scope.deploymentConfigId || $scope.deploymentConfigMissing) {
+            // Scale as replication controller.
+            return true;
+          }
+
+          if (!$scope.deploymentConfig) {
+            // Wait for the deployment config.
+            return false;
+          }
+
+          // Only allow scaling of the latest deployment.
+          return deploymentIsLatestFilter($scope.rc, $scope.deploymentConfig);
```

Comment:
I have some changes locally to find the most recent in progress or complete deployment to enable scaling controls. This is probably the best we can do now short of redesigning the overview.

<img width="640" alt="openshift_web_console" src="https://cloud.githubusercontent.com/assets/1167259/11251732/9baa5562-8e00-11e5-990b-671b22d2ba01.png">